contracts/src/v0.8/dev/interfaces/OCR2DRRegistryInterface.sol

```diff
@@ -0,0 +1,175 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.6;
+
+/**
+ * @title OCR2DR billing subscription registry interface.
+ */
+interface OCR2DRRegistryInterface {
+  struct RequestBilling {
+    address client;
+    // a unique subscription ID allocated by billing system,
+    uint64 subscriptionId;
+    // customer specified gas limit for the fulfillment callback
+    uint32 gasLimit;
+  }
+
+  /**
+   * @notice Get configuration relevant for making requests
+   * @return uint32 global max for request gas limit
+   * @return address[] list of registered DONs
+   */
+  function getRequestConfig() external view returns (uint32, address[] memory);
+
+  /**
+   * @notice Determine the charged fee that will be paid to the Registry owner
```

Comment:
Whos is the "Registry owner"? Is this CLL & the means by which CLL is paid for providing this technology?

Reply:
Yes exactly, CLL.